class-wxr-import-ui.php

```diff
@@ -464,12 +467,22 @@ public function stream_import() {
 		$mapping = $settings['mapping'];
 		$this->fetch_attachments = (bool) $settings['fetch_attachments'];
 
-		$importer = $this->get_importer();
-		if ( ! empty( $mapping['mapping'] ) ) {
-			$importer->set_user_mapping( $mapping['mapping'] );
-		}
-		if ( ! empty( $mapping['slug_overrides'] ) ) {
-			$importer->set_user_slug_overrides( $mapping['slug_overrides'] );
+		// Try to restore the importer from a previous run.
+		$importer = get_post_meta( $this->id, '_importer_state', true );
```

Comment:
It essentially is option, see the "Resume" intermediary screen, however I need to add the delete resume data from that.